runtime/doc/options.txt

```diff
@@ -2870,6 +2869,16 @@ A jump table for the options with a short description can be found at |Q_op|.
 	apply equally to 'grepprg'.
 	This option cannot be set from a |modeline| or in the |sandbox|, for
 	security reasons.
+	This option defaults to `findstr /n $* nul` on Windows
+	and `grep -n $* /dev/null` elsewhere.
+	Additionally, if ripgrep is available (|executable()|) then this option
+	and |'grepformat'| default to `rg --vimgrep -uuu ...` and `%f:%l:%c:%m` respectively.
```

Comment:
I think listing the order of tools in reverse priority extremely unintuitive. I think it's better to present it like `:h clipboard-tool` to eliminate any ambiguities. Something like (adjust as needed):

  - ripgrep
  - findstr (Windows)
  - grep (Unix)

Reply:
some structure in the text would also be helpful.